app/src/main/java/com/napzak/market/core/designsystem/component/button/EnableDisableTextButton.kt

```diff
@@ -0,0 +1,65 @@
+package com.napzak.market.core.designsystem.component.button
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.napzak.market.core.designsystem.theme.NapzakMarketTheme
+
+/**
+ * 활성화/비활성화 버튼 컴포넌트
+ *
+ * @param text 버튼에 표시될 텍스트
+ * @param isEnabled 버튼 활성화 여부
+ * @param onClick 버튼 클릭 시 수행할 동작
+ */
+
+@Composable
+fun EnableDisableTextButton(
+    text: String,
+    isEnabled: Boolean,
+    onClick: () -> Unit
+) {
+    CommonButton(
+        text = text,
+        onClick = { if (isEnabled) onClick() },
+        isEnabled = isEnabled,
+        width = 335.dp,
+        height = 52.dp,
+        shape = RoundedCornerShape(12.dp),
+        textStyle = NapzakMarketTheme.typography.bodyBold16,
+        backgroundColor = NapzakMarketTheme.colors.purple30,
+        contentColor = NapzakMarketTheme.colors.white
+    )
+}
```

Comment:
컴포넌트의 디자인을 지정하는 속성들도 여러 화면에서 다르게 쓰일 상황을 대비하여 생성자에 선언하도록 합시다!
만약 자주 쓰이는 값이 있다면 기본값으로 설정해서 코드 수를 줄일 수 있습니다!!
```suggestion
@Composable
fun EnableDisableTextButton(
    text: String,
    isEnabled: Boolean,
    onClick: () -> Unit,
    shape: Shape = RoundedCornerShape(12.dp),
    textStyle: TextStyle = defaultTypography.bodyBold16,
    backgroundColor: Color = Purple30,
    contentColor: Color = White
) {
    CommonButton(
        text = text,
        onClick = { if (isEnabled) onClick() },
        isEnabled = isEnabled,
        width = 335.dp,
        height = 52.dp,
        shape = shape,
        textStyle = textStyle,
        backgroundColor = backgroundColor,
        contentColor = contentColor,
    )
}
```